index.html

```diff
@@ -221,36 +229,36 @@ <h2>
                     <button class="contact-btn btn-anime" type="submit">Get in touch</button>
                 </form>
             </div>
-             <div class="socials2">
-                <ul class="socials">
-                    <li>
-                        <a href="https://twitter.com/precious_akams">
-                        <img src="icons/twitter.png" alt="twitter_link">
-                     </a>
-                     </li>
-                    <li>
-                        <a href="https://www.linkedin.com/in/onyeagoziri-akams/">
-                        <img src="icons/Linkedin.png" alt="linkedin_link">
-                     </a>
-                    </li>
-                    <li>
-                     <a href="https://medium.com/@kamzzytech">
-                         <img src="icons/medium.png" alt="medium_link">
-                      </a>
-                    </li>
-                    <li>
-                        <a href="https://github.com/kamzzy">
-                        <img src="icons/github.png" alt="github_link">
-                     </a>
-                     </li>
-                    <li>
-                        <a href="https://angel.co/u/onyeagoziri-akams">
-                        <img src="icons/angellist.png" alt="angellist_link">
-                     </a>
-                     </li> 
-                 </ul>
             </div>
         </section>
+        <ul class="socials social2">
+            <li>
+                <a href="https://twitter.com/precious_akams">
+                <img src="icons/twitter.png" alt="twitter_link">
+             </a>
+             </li>
+            <li>
+                <a href="https://www.linkedin.com/in/onyeagoziri-akams/">
+                <img src="icons/Linkedin.png" alt="linkedin_link">
+             </a>
+            </li>
+            <li>
+             <a href="https://medium.com/@kamzzytech">
+                 <img src="icons/medium.png" alt="medium_link">
+              </a>
+            </li>
+            <li>
+                <a href="https://github.com/kamzzy">
+                <img src="icons/github.png" alt="github_link">
+             </a>
+             </li>
+            <li>
+                <a href="https://angel.co/u/onyeagoziri-akams">
+                <img src="icons/angellist.png" alt="angellist_link">
+             </a>
```

Comment:
- [ ] Kindly add this vertical line before your social icons

- Your Design

![55555555](https://user-images.githubusercontent.com/79058364/161379486-a8fc6b1f-4daa-4f08-8602-d06792a90aa1.png)

- Figma design

![go5555](https://user-images.githubusercontent.com/79058364/161379500-7e3c335c-ade7-48aa-9e19-270cfb6dfbab.png)